### ISSUE TITLE
Feature request: support "Capital pitch change"

### ISSUE THREAD

**Issue by cary-rowen** [leonardder/rdAccess#10]
## Steps to reproduce:
1. Make sure the "Capital pitch change percentage" is set to 30 on both the server and the client.
2. On the server, open notepad and try typing uppercase letters and then try typing lowercase letters.
## Actual performance
NVDA can't distinguish capital letters by pitch change
## Expected performance
NVDA should use speech synthesizer pitch changes to indicate capital letters.

**Comment by leonardder** [comment]
Could you please test this with Espeak? As it works okay on my end. I'm afraid it might again be the custom synth you're using.

**Comment by cary-rowen** [comment]
Hi, 
I tested with ESpeak and confirmed that ESpeak can reproduce this, but Windows OneCore works as expected.
I'm using the latest alpha version of NVDA installed.
Thanks

**Comment by leonardder** [comment]
In the situation where it doesn't work, could you please provide the result of the following python console command on the remote system?
```
import synthDriverHandler
synth = synthDriverHandler.getSynth()
synth.supportedCommands
```

**Comment by leonardder** [comment]
have you disabled the option `Allow remote system to control driver settings` by any chance? Disabling that option causes capital pitch change to fail. I created https://github.com/nvaccess/nvda/issues/15262

**Comment by cary-rowen** [comment]
@leonardder 
> have you disabled the option Allow remote system to control driver settings by any chance? Disabling that option causes capital pitch change to fail. I created  nvaccess/nvda#15262 

Yes, I enabled the option 'Capital pitch change' and it worked as expected.
If this is known to be the case, you can close this.

Thanks

**Comment by leonardder** [comment]
I think I should document this as a known issue before closing.
